editor/svgcanvas.js

```diff
@@ -2254,28 +2263,29 @@ const mouseMove = function (evt) {
       dy = r * Math.sin(theta);
     }
 
+    var normalized_dx = dx, normalized_dy = dy;
```

Comment:
Please use `const` where possible and `let` otherwise (and please run `npm run eslint` to make sure your PR doesn't add any new linting issues such as this).

Reply:
Thank you for the feedback. While we are on the topic of code quality, was there an area where documentation was lacking? I will try to sprinkle some comments into my revisions where they're appropriate.

Reply:
You could add mention of your addition to `docs/tutorials/Editor.md`, and this file could in general be improved as well (in a separate PR), as there is precious little aimed at users uninitiated in graphics editors to svgedit.

In this besides a mention of the key-sequence and behavior in `Editor.md`, it'd be ideal if you could also add a ui test, as that is, AFAIC, the gold standard for testing the app, even while unit tests are still useful.

Reply:
I don't think that improvements to `Editor.md` fit into the scope of this PR, but it'd be something to look at. Since that file is so slim right now, it would take a lot of groundwork to explain each tool that the ctrl-click applies to, so I'll leave this out of this PR.

Reply:
General improvements would not fit into scope, but I would like to enforce a policy of documenting any new UI features, so I would appreciate it if you'd just describe the functionality you added/changed.

You don't have to leave the groundwork there for the tools (except maybe to add a "(To-do)" block in the sections you think need addition. Having these updated docs should both help encourage us to do this gradually, and be there to inform anyone familiar with older behavior (e.g., if the earlier developers were to resume work).

Likewise if you can add a Testcafe test--of if you need help with it, give me a human-readable series of steps here (with screenshots if necessary), and I can try to provide the Testcafe code. (Though I may switch us to Cypress as they know have UI testing code coverage, which is really awesome.) I am not even into design myself, so I should be a good sounding board to ensure the steps are clear.

Actually, the tests are even more important than docs, not only for regressions, but also as it is itself a form of documentation.